lib/active_model/serializer/caching.rb

```diff
@@ -89,6 +89,10 @@ def fragmented(serializer)
         # https://github.com/rails-api/active_model_serializers/pull/1249#issuecomment-146567837
         def cache(options = {})
           self._cache = ActiveModelSerializers.config.cache_store if ActiveModelSerializers.config.perform_caching
```

Comment:
easiest impl, have a class attribute `perform_caching` that acts as the sentinel. That was we don't have to change the api much... caching can still be turning on/off per serializer, but configured at load and disabled/enabled before use